dhall-json/src/Dhall/JSON.hs

```diff
@@ -624,12 +624,12 @@ parseOmission :: Parser (Value -> Value)
 parseOmission =
         Options.Applicative.flag'
             omitNull
-            (   Options.Applicative.long "omitNull"
+            (   Options.Applicative.long "omit-null"
             <>  Options.Applicative.help "Omit record fields that are null"
             )
     <|> Options.Applicative.flag'
             omitEmpty
-            (   Options.Applicative.long "omitEmpty"
+            (   Options.Applicative.long "omit-empty"
```

Comment:
What about the move from `--omitEmpty` to `--omissible-lists` discussed in https://github.com/dhall-lang/dhall-haskell/pull/1414 and originally proposed in https://github.com/dhall-lang/dhall-haskell/issues/1410?

Would you mind checking that `--omitEmpty` and `--omissible-lists` are indeed opposites?

Reply:
`omit-mpty` also "nullifies" not only empty arrays but also empty objects. `omissible-lists` translates only `null ~> []` but not `null ~> {}`. So `omit-empty` and `omissible-lists` are not complete opposites.

Reply:
Thanks for checking! Seems like a bit of an unfortunate oversight! Let's maybe keep the `--omit-empty` name for now then, and discuss this further on the issue.